### DISCUSSION TITLE
Need some advices

### DISCUSSION THREAD

**Discussion by odil24** [GPUOpen-LibrariesAndSDKs/RadeonProRenderBlenderAddon#496] [Q&A]
@VascoPi Hi. If you have an email plz send here. I need some advices.

**Comment by VascoPi** [comment]
You're already at the right place to ask questions. To get some assistance, could you please reveal your inquiry?

**Reply by odil24** [reply]
> You're already at the right place to ask questions. To get some assistance, could you please reveal your inquiry?

The question isnt related to RPR only for my addon. Addon is called BCorona (Corona for Blender exporter). It does not supported yet Modifiers, Auto-smooth shading and Particles system. I know it is related to Depsgraph but I really still confusing via DEG. I saw many engines: LuxCore, RPR, Renderman...but nothing understand.

**Reply by bsavery** [reply]
I think your question might be more appropriate to ask on https://devtalk.blender.org
